charts/tempo-distributed/templates/metamonitoring/kubelet-cadvisor-servmon.yaml

```
@@ -0,0 +1,83 @@
+{{- if and ((.Values.metaMonitoring).grafanaAgent).enabled ((((.Values.metaMonitoring).grafanaAgent).metrics).scrapeK8s).enabled }}
```

same as above, why not:
```suggestion
{{- if and ((.Values.metaMonitoring).grafanaAgent).enabled .Values.metaMonitoring.grafanaAgent.metrics.scrapeK8s.enabled }}
```